src/components/common.js

```diff
@@ -65,111 +65,114 @@ const search = function({ wrapperProps = {}, inputProps = {} }) {
     ])
 }
 
-const topBar = children => h(TopBarConstructor, children)
-
-class TopBarConstructor extends Component {
-  constructor(props) {
-    super(props)
-    this.state = { navShown: false }
-  }
+export function topBar(title, children) {
+  return h(class topBar extends Component {
```

Comment:
Exporting a function like this means that our components are called in the same way as non-custom components, and can still be documented & show up with the same component name in code and in the react devtools.